be/test/storage/delete_handler_test.cpp

```diff
@@ -78,7 +78,7 @@ void tear_down() {
 }
 
 void set_default_create_tablet_request(TCreateTabletReq* request) {
-    request->tablet_id = 10003;
+    request->tablet_id = random();
```

Comment:
why to change the random()

Reply:
Cannot reuse the same tablet id in test cases, otherwise the `create_tablet()` will fail.